selectors.go

```diff
@@ -9,15 +9,33 @@ import (
 
 //go:embed selectors.yml
 var selectorsYml []byte
-var evmChainIdToChainSelector = parseYml()
 
-func parseYml() map[uint64]uint64 {
+//go:embed test_selectors.yml
+var testSelectorsYml []byte
+
+var evmChainIdToChainSelector = loadAllSelectors()
+
+func loadAllSelectors() map[uint64]uint64 {
+	selectors := parseYml(selectorsYml)
+	testSelectors := parseYml(testSelectorsYml)
+
+	output := make(map[uint64]uint64, len(selectors)+len(testSelectors))
+	for k, v := range selectors {
+		output[k] = v
+	}
+	for k, v := range testSelectors {
```

Comment:
I think we'd want to ensure these are distinct sets right? ie panic on set overlap

Reply:
Makes sense, will do this defensive check.